packages/host/app/components/search-sheet/search-result/index.gts

```diff
@@ -0,0 +1,43 @@
+import Component from '@glimmer/component';
+import type { Card } from 'https://cardstack.com/base/card-api';
+import { cardTypeDisplayName } from '@cardstack/runtime-common';
+import { CardContainer } from '@cardstack/boxel-ui';
+import { trackedFunction } from 'ember-resources/util/function';
+import type CardService from '../../../services/card-service';
+import { service } from '@ember/service';
+
+interface Signature {
+  Element: HTMLElement;
+  Args: {
+    card: Card;
+    index: number; //currently this arg is only for testing purposes
+  };
+}
+
+export default class SearchResult extends Component<Signature> {
+  <template>
+    <CardContainer @displayBoundaries={{true}} class='search-result' data-test-search-result-index={{@index}} data-test-search-result={{@card.id}}>
```

Comment:
Instead of passing `@index` to this component, use `...attributes` instead and specify `data-test-search-result-index={{index}}` when invoking the component.